app/models/work.rb

```diff
@@ -53,6 +53,10 @@ def already_embargo_released?
     head&.deposited? && head.embargo_date.present? && head.embargo_date <= Time.zone.today
   end
 
+  def embargoed?
+    head&.deposited? && head.embargo_date.present? && head.embargo_date > Time.zone.today
```

Comment:
Why do we need the null test on head here? There should be no way to have created a record in that state.

Reply:
I don't know, but the existing `already_immediately_released?` and `already_embargo_released?` have the null test. I was just copying.

Reply:
Yes, I believe this is required for those because they are used on the works/new form.  If this is not used in that scenario, it isn't required.